deal.II-toolchain/packages/petsc.package

```diff
@@ -104,7 +121,7 @@ package_specific_setup () {
 
 package_specific_register () {
     export PETSC_DIR=${INSTALL_PATH}
-    if [ ! -z "${PARMETIS_DIR}" ]; then
+    if [ ! -z "${PARMETIS_DIR}" ] && [ -z "${SCALAPACK_DIR}" ]; then
```

Comment:
Why do we need to check for parmetis?

Reply:
Again the same answer as above: I have not tested whether this check is necessary, I just assumed that it is necessary because it was already there.